Provider/ImageProvider.php

```diff
@@ -72,12 +72,32 @@ public function getHelperProperties(MediaInterface $media, $format, $options = a
             $box = $this->resizer->getBox($media, $resizerFormat);
         }
 
+        $mediaWidth = $box->getWidth();
+
+        $srcset = array();
```

Comment:
Please use `srcSet`, it will be clearer for dummies like me who encounter this for the first time.